pages/internship.js

```diff
@@ -0,0 +1,147 @@
+import Link from 'next/link';
+import React from 'react';
+
+import Faq from 'react-faq-component';
+import Carousel from 'react-multi-carousel';
+
+import Banner from '../components/Banner';
+import CommitteeInternshipInfo from '../components/Internship/CommitteeInternshipInfo';
+import InternshipTimeline from '../components/Internship/InternshipTimeline';
+import NextSteps from '../components/Internship/NextSteps';
+import Layout from '../components/Layout';
+
+import data from '../data';
+import 'react-multi-carousel/lib/styles.css';
+
+const { committees, internship } = data;
+committees.unshift(
```

Comment:
Quick note - try not to use `committees.unshift`, since you're modifying the array in-place (and I think causing some other shenanigans - try going to the internship page, and then switching to the committees page. I get an error).

Instead, I suggest concatenating it, which may also involve deep-copying:

```js
const boardInfo = {...};
const internshipInfo = boardInfo.concat(committees);
```

More on `concat` in the [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat).